public/assets/plugins/file.min-49f21da99c66e1ddc7de81f4af1bbf812122bd588cb97731f243870482ec5ffa.js

```diff
@@ -0,0 +1,8 @@
b.button.buildList(b.opts.fileInsertButtons)+"</div>";var c="";c='<div class="fr-file-upload-layer fr-layer fr-active" id="fr-file-upload-layer-'+b.id+'"><strong>'+b.language.translate("Drop file")+"</strong><br>("+b.language.translate("or click")+')<div class="fr-form"><input type="file" name="'+b.opts.fileUploadParam+'" accept="/*" tabIndex="-1"></div></div>';var d='<div class="fr-file-progress-bar-layer fr-layer"><h3 class="fr-message">Uploading</h3><div class="fr-loader"><span class="fr-progress"></span></div><div class="fr-action-buttons"><button type="button" class="fr-command" data-cmd="fileDismissError" tabIndex="2">OK</button></div></div>',e={buttons:a,upload_layer:c,progress_bar:d},f=b.popups.create("file.insert",e);return b.popups.onHide("file.insert",q),p(f),f}function s(c){return a(c).hasClass("fr-file")?b.events.trigger("file.unlink",[c]):void 0}function t(){var c=function(a){a.preventDefault()};b.events.on("dragenter",c),b.events.on("dragover",c),b.events.on("drop",function(c){b.popups.hideAll();var e=c.originalEvent.dataTransfer;if(e&&e.files&&e.files.length){var f=e.files[0];if(f&&f.type&&(b.opts.fileAllowedTypes.indexOf(f.type)>=0||b.opts.fileAllowedTypes.indexOf("*")>=0)){b.markers.insertAtPoint(c.originalEvent),b.markers.remove(),b.popups.hideAll();var g=b.popups.get("file.insert");g||(g=r()),b.popups.setContainer("file.insert",a(b.opts.scrollableContainer)),b.popups.show("file.insert",c.originalEvent.pageX,c.originalEvent.pageY),d(),o(e.files),c.preventDefault(),c.stopPropagation()}}})}function u(){b.events.disableBlur(),b.selection.restore(),b.events.enableBlur(),b.popups.hide("file.insert"),b.toolbar.showInline()}function v(){t(),b.events.on("link.beforeRemove",s)}var w=1,x=2,y=3,z=4,A=5,B=6,C=7,D={};return D[w]="File cannot be loaded from the passed link.",D[x]="No link in upload response.",D[y]="Error during file upload.",D[z]="Parsing response failed.",D[A]="File is too large.",D[B]="File file type is invalid.",D[C]="Files can be uploaded only to same domain in IE 8 and IE 9.",{_init:v,showInsertPopup:c,upload:o,insert:h,back:u,hideProgressBar:e}},a.FroalaEditor.DefineIcon("insertFile",{NAME:"file-o"}),a.FroalaEditor.RegisterCommand("insertFile",{title:"Upload File",undo:!1,focus:!0,refershAfterCallback:!1,popup:!0,callback:function(){this.popups.isVisible("file.insert")?(this.$el.find(".fr-marker")&&(this.events.disableBlur(),this.selection.restore()),this.popups.hide("file.insert")):this.file.showInsertPopup()}}),a.FroalaEditor.DefineIcon("fileBack",{NAME:"arrow-left"}),a.FroalaEditor.RegisterCommand("fileBack",{title:"Back",undo:!1,focus:!1,back:!0,refreshAfterCallback:!1,callback:function(){this.file.back()},refresh:function(a){this.opts.toolbarInline?(a.removeClass("fr-hidden"),a.next(".fr-separator").removeClass("fr-hidden")):(a.addClass("fr-hidden"),a.next(".fr-separator").addClass("fr-hidden"))}}),a.FroalaEditor.RegisterCommand("fileDismissError",{title:"OK",callback:function(){this.file.hideProgressBar(!0)}})});
```

Comment:
Line is too long.<br>Missing semicolon.<br>Expected an identifier and instead saw '=='.<br>Expected an operator and instead saw 'typeof'.<br>Expected an assignment or function call and instead saw an expression.<br>Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.<br>Mixed double and single quotes.<br>'c' is already defined.<br>Confusing use of '!'.<br>Expected '{' and instead saw 'return'.<br>Possible strict violation.<br>Expected '{' and instead saw 'if'.<br>Expected '{' and instead saw 'n'.<br>'k' is already defined.<br>Too many errors. (88% scanned).